cmeutils/tests/test_plot.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+import scipy.signal as signal
+import numpy as np
+
+from base_test import BaseTest
+
+from cmeutils.plot_tools import find_peaks
+
+class TestPlot(BaseTest):
+    def test_find_peaks(self, rdf_txt):
+        line= np.genfromtxt(rdf_txt, names=True, delimiter=",")
+        y= line["rdf"]
+        peaks = signal.find_peaks(y)
+        assert isinstance(y, np.ndarray)
+        assert isinstance(peaks, tuple)
```

Comment:
Nice, now last step just check that a couple of peaks are in the right place. I do wonder if `scipy.signal.find_peaks` is used with no arguments (the default of your function), if any peaks will be found... I would plot this to make sure it makes sense :)